### DISCUSSION TITLE
Possibleto pass configuration Settings to dotnet test?

### DISCUSSION THREAD

**Discussion by groogiam** [xunit/xunit#2566] [Question] [answered]
Is there a way to pass the configuration parameters documented in https://xunit.net/docs/configuration-files to the test framework using  dotnet test via the command line? 

I tried some thing like `dotnet test -- xunit.maxParallelThreads=1` but this does not seem to work.

**Comment by bradwilson** [comment, marked as answer]
No, these can only be set via configuration file.
